lib/bot.js

```diff
@@ -208,6 +208,11 @@ class Bot {
         this.ircClient.say(ircChannel, prelude);
       } else if (!message.subtype) {
         text = `<${user.name}> ${text}`;
+      } else if (message.subtype === 'file_share') {
+        text = `<${user.name}> File uploaded ${message.file.permalink} / ${message.file.permalink_public}`;
```

Comment:
permalink_public is always generated even if file is not shared. So it's rather hard to guess which would be better to show.